packages/form-core/src/validate/Validator.js

```diff
@@ -111,6 +128,38 @@ export class Validator {
    * - Or, when a webworker was started, its process could be aborted and then restarted.
    */
   abortExecution() {} // eslint-disable-line
+
+  __fakeExtendsEventTarget() {
```

Comment:
Would it be the right moment to abstract this throughout the whole codebase?

Have the feeling that tech debt increases as this gets more verbose

Reply:
We want to get rid of this eventually right? We only do it in two places, here and in overlay controller I believe (or mixin..). For me personally, I don't like to abstract until we get 3 instances and now we have two. Also, abstracting this is really really tough for typing, which is one of the reasons why I added it here inside the class instead of as an external util. The reason why it's tough to type when you add these methods through an external method, is because you need to somehow tell this class that those methods (addEventListener etc.) have now become available because we called fakeExtendsEventTarget in the constructor. To my knowledge there was no elegant way to do that unless we make the fakeExtendsEventTarget a mixin so the methods can be inherited.

Reply:
I think we use it in 4 places throughout our code base (overlay, validate, localize, form registration).
Let's fix it in a later pr 👍